server/routes/debug.ts

```diff
@@ -89,14 +90,13 @@ export default function makeDebugRoutes(services: Services): Record<string, Requ
     },
 
     async incidentDetails(req, res) {
-      const { userService } = services
-      const { incidentReportingApi, prisonApi } = res.locals.apis
-
       const { id } = req.params
       if (!id) {
         throw new NotFound()
       }
 
+      const { incidentReportingApi, prisonApi } = res.locals.apis
```

Comment:
Actually, I've purposefully put this up at the beginning of the function as this are kinda the dependencies this function depends on, rather then scattering these around the function.